metrics_ci_integration/test/jenkins/client/model/jenkins_build_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:ci_integration/jenkins/client/model/jenkins_build.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("JenkinsBuild", () {
+    final currentMillisecondsSinceEpoch = DateTime.now().millisecondsSinceEpoch;
```

Comment:
```suggestion
    final currentTimestamp = DateTime.now();
```
Can we replace this by DateTime timestamp to make the code a bit more readable?